site/docs/getting-started.mdx

```diff
@@ -4,8 +4,102 @@ sidebar_position: 1
 
 # Getting Started
 
-Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+## Developing
 
-## Lorem ipsum
+Learn how to use Odyssey's [UI components](/docs/components/) in your React web applications.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+### Pre-requisites
+
+In order to use our components you will need:
+
+- Familiarity with web development fundamentals (HTML, CSS & JavaScript) and [React](https://reactjs.org/)
+- [Node.js](https://nodejs.org/) v16 or later installed on your system.
+- A React (v16 or later) web application project.
+  - For new projects, you can use [`create-react-app`](https://github.com/facebook/create-react-app#creating-an-app) or similar tools.
+
+### Install packages
+
+You will need to add the following three dependencies to your project:
+
+```sh
+npm install --save @jpmorganchase/uitk-core @jpmorganchase/uitk-theme @jpmorganchase/uitk-icons
+```
+
+:::note
+
+If you prefer, you can use [`yarn`](https://yarnpkg.com/) or other package managers to install our NPM packages.
```

Comment:
Docusarus light mode links are really hard to see

Reply:
The current styling is just temporary. I'll ask the team to ensure we eventually choose a link color that is easier to distinguish from surrounding text while still having a sufficient contrast ratio against the background.

Reply:
Yeah, I figured. Just wanted to mention it. Dark mode is fine though :)